wire-ios/Tests/Extensions/E2eIdentityCertificate+Mocks.swift

```diff
@@ -18,13 +18,26 @@
 
 import Foundation
 
+extension String {
```

Comment:
Did you delete the source `wire-ios-data-model/Source/Utilis/String+mocks.swift`, because this was my original concern to have it in production?

Reply:
https://github.com/wireapp/wire-ios/pull/945#discussion_r1495819465

Reply:
I added it to test target.